### PR TITLE
Suggestions for BEP-009

### PR DESCRIPTION
**Pull request by effigies**
This PR aims to be a series of self-contained commits that may be reviewed individually by BEP 009 leads and accepted or rejected. For example f66ff90d4265ef0c04eae846abac8083960c48f3 modifies the MRI portion of the spec to indicate that a piece of metadata is required in MR data if a dataset contains PET data.

I will continue adding to this as I see to it. BEP-009 leads are welcome to merge or cherry-pick at any time. I will take care of any conflicts.

### CODE REVIEW

**Comment by effigies** [comment]
cc @mnoergaard @melanieganz

**Comment by effigies** [comment]
I am unlikely to be able to work much more on this before Tuesday, Jan 19, so please feel free to review these commits before then.

**Comment by sappelhoff** [comment]
It may be worth it to rebase this PR on `master`, now that ASL has been merged. There are a few conflicts:

```
Unmerged paths:
  (use "git add <file>..." to mark resolution)

	both modified:   mkdocs.yml
	both modified:   src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```

**Review by mnoergaard — APPROVED** [review]

**Review by mnoergaard — APPROVED** [review]

**Comment by sappelhoff** [comment]
@mnoergaard should we merge this? Or do you want to wait for more reviews?

File: src/02-common-principles.md
Comment (mnoergaard) [inline src/02-common-principles.md:84]: Not strictly correct wording, as the injection may start at time X, but the participant will not necessarily be entering the scanner (i.e. acquisition start) until time Y.

```suggestion
    for some types of acquisition (dynamic PET), the subject may leave and re-enter the scanner without interrupting the scan. 
```

https://link.springer.com/chapter/10.1007%2F3-540-48714-X_8

File: src/04-modality-specific-files/09-positron-emission-tomography.md
Comment (mnoergaard) [inline src/04-modality-specific-files/09-positron-emission-tomography.md:39]:
```suggestion
    injection may in some cases be considered separate sessions.
```

File: src/04-modality-specific-files/09-positron-emission-tomography.md
Comment (mnoergaard) [inline src/04-modality-specific-files/09-positron-emission-tomography.md:41]: I wouldn't say that it is common, but that it may happen (some times a scan may last for 120 minutes). 

```suggestion
1.  **Run** - In PET, subjects may have to leave the scanner to use the
```

File: src/04-modality-specific-files/09-positron-emission-tomography.md
Comment (mnoergaard) [inline src/04-modality-specific-files/09-positron-emission-tomography.md:109]:
```suggestion
If multiple reconstructions of the data are made with the same type of reconstruction,
```

**Review by mnoergaard — CHANGES_REQUESTED** [review]

File: src/04-modality-specific-files/09-positron-emission-tomography.md
Comment (mnoergaard) [reply src/04-modality-specific-files/09-positron-emission-tomography.md:109]: @sappelhoff. I made som additional changes. I would like to wait for others to review it before merging. @melanieganz

File: src/04-modality-specific-files/09-positron-emission-tomography.md
Comment (effigies) [reply src/04-modality-specific-files/09-positron-emission-tomography.md:39]: I meant "logically" in terms of "for the purposes of analysis" (similar to a "logical partition" instead of "physical partition" in disk drives). I think I would prefer to just drop "logically".

File: src/04-modality-specific-files/09-positron-emission-tomography.md
Comment (effigies) [reply src/04-modality-specific-files/09-positron-emission-tomography.md]: Applied.

File: src/02-common-principles.md
Comment (effigies) [reply src/02-common-principles.md]: For simplicity went with:

> For some types of [PET](04-modality-specific-files/09-positron-emission-tomography.md),
> a subject may leave and re-enter the scanner without interrupting the scan.

If additional text would be useful here, please let me know.

**Review by mnoergaard — APPROVED** [review]

File: src/02-common-principles.md
Comment (melanieganz) [reply src/02-common-principles.md]: Maybe I would suggest adding the term acquisition, so: For some types of PET acquisitions, a subject may leave and re-enter the scanner without interrupting the scan.

File: src/02-common-principles.md
Comment (melanieganz) [inline src/02-common-principles.md:83]:
```suggestion
    For some types of [PET](04-modality-specific-files/09-positron-emission-tomography.md) acquisitions,
```

**Review by melanieganz — APPROVED** [review]
Thanks @effigies  and @sappelhoff for the detailed changes. I agree with your suggestions, so this can be merged and should be merged before we continue with the rest of the comments that were added later on by @robertoostenveld .

**Pull request merged**
